experiments/standalone/Soil/richards_runoff.jl

```diff
@@ -168,7 +168,13 @@ drivers = ClimaLand.get_drivers(model)
 updatefunc = ClimaLand.make_update_drivers(drivers)
 driver_cb = ClimaLand.DriverUpdateCallback(updateat, updatefunc)
 cb = SciMLBase.CallbackSet(driver_cb, saving_cb)
-sol = @time SciMLBase.solve(prob, ode_algo; dt = dt, saveat = dt, callback = cb)
+sol = @time SciMLBase.solve(
+    prob,
+    ode_algo;
+    dt = dt,
+    saveat = saveat,
```

Comment:
I think this is failing because the saveat is at every 100 dt, but we take < 100 steps. We can make saveat =[t0,tf] here.